packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/Button/Button.js

```diff
@@ -68,19 +70,21 @@ class Button extends React.Component {
 
     const hoveredOrFocused = hovered || focused
 
-    const eventHandlers = {
-      onBlur,
-      onClick,
-      onFocus,
-      onMouseOver,
-      onMouseOut
-    }
+    const eventHandlers = (disabled)
+      ? {}
+      : {
+          onBlur,
+          onClick,
+          onFocus,
+          onMouseOver,
+          onMouseOut
+        }
 
     const childrenWithProps = React.Children.map(children, child =>
       React.cloneElement(child, { ...this.getSize(size) }))
 
     return (
-      <StyledButton {...eventHandlers}>
+      <StyledButton {...eventHandlers} disabled={disabled ? 'true' : ''}>
```

Comment:
This might be throwing the warning `Received the string "true" for the boolean attribute "disabled". Although this works, it will not work as expected if you pass the string "false". Did you mean disabled={true}?`? Not blocking though.

Reply:
Naa, I'll fix that - thanks